Sprint-1/stretch/aoc-2018-day1/solution.js

```diff
@@ -0,0 +1,24 @@
+const fs = require('fs');
+const path = require('path');
+// Function to calculate the resulting frequency
+const calculateFrequency = (changes) => {
+    let frequency = 0; // Start at zero
+    for (let change of changes) 
+        frequency += parseInt(change, 10); 
+    return frequency;
+};
+
+const inputFilePath = path.join(__dirname, 'input.txt');
+let puzzleInput;
+
+try {
+    const fileContent = fs.readFileSync(inputFilePath, 'utf-8');
+    puzzleInput = fileContent.split(/\r?\n/).filter(line => line.trim() !== ''); 
```

Comment:
Why split the input with this specific pattern? Why not just split using the newline character "\n"?

Reply:
Yes, You are correct we can use "\n" this is the easy way.

Thank you